internal/context/ue.go

```diff
@@ -119,7 +122,7 @@ func (ue *UeContext) NewUeAMPolicyData(assolId string, req models.PolicyAssociat
 		Rfsp:              req.Rfsp,
 		Guami:             req.Guami,
 		UserLoc:           req.UserLoc,
-		ServiveName:       req.ServiveName,
+		ServiceName:       req.ServiceName,
```

Comment:
According to Newest openapi hash, you'll meet this error: `ServiceName: req.ServiveName` for type PcfAmPolicyControlPolicyAssociationRequest struct
So maybe you can create a new PR to fix the new OpenAPI in models/model_pcf_am_policy_control_policy_association_request.go

Reply:
![image](https://github.com/user-attachments/assets/877efe87-43c6-43f6-ad80-7f2e92d07915)
I found out that OpenAPI file is correct.